src/config/config_service.py

```diff
@@ -122,6 +131,33 @@ def update_config(self, user, config, filename, uploaded_script):
         LOGGER.info('Updating script config "' + name + '" in ' + original_file_path)
         self._save_config(config, original_file_path)
 
+
+    def delete_config(self, user, name):
+        self._check_admin_access(user)
+
+        search_result = self._find_config(name, user)
+        if search_result is None:
+            raise InvalidConfigException(f'Config with the name "{name}" not found')
+
+        (short_config, path, config_object) = search_result
+
+        if not self._can_edit_script(user, short_config):
+            raise ConfigNotAllowedException(
+                f'{str(user)} has no admin access to {short_config.name}'
+            )
+
+        archived_configs_folder = os.path.join(self._script_configs_folder, '..', 'deleted')
+        file_utils.prepare_folder(archived_configs_folder)
+
+        archive_file_name = _add_date_prefix_to_filename(os.path.basename(path))
```

Comment:
You should also change an extension to be different from `json`, otherwise script server will find this configuration.
I think it could be either `._json` or `.json.deleted`

Reply:
ah yes you are right, i forgot about this